account_check_printing_report_base/report/lang.py

```diff
@@ -2,22 +2,22 @@
 from num2words import num2words
 from num2words import CONVERTER_CLASSES, CONVERTES_TYPES
 from num2words.lang_ES import Num2Word_ES
+from num2words.base import Num2Word_Base
 
 num2words_by_lang = {
     'es': 'Num2WordESCustom'
 }
 
 
 class Num2WordESCustom(Num2Word_ES):
-    # Replace centavo to céntimo. Original method copy
+    CURRENCY_FORMS = {
+        'EUR': (('euro', 'euros'), ('céntimo', 'céntimos')),
+    }
+
+    # TODO: PR to remove overwrite in num2words code and use CURRENCY_FORMS
     def to_currency(self, val, longval=True, old=False):
-        hightxt, lowtxt = "euro/s", u"céntimo/s"
-        if old:
-            hightxt, lowtxt = "peso/s", "peseta/s"
-        result = self.to_splitnum(val, hightxt=hightxt, lowtxt=lowtxt,
-                                  divisor=1, jointxt="y", longval=longval)
-        # Handle exception, in spanish is "un euro" and not "uno euro"
-        return result.replace("uno", "un")
+        return Num2Word_Base.to_currency(self, val, currency='EUR', cents=True,
```

Comment:
Wasn't the problem to use this to have the word `centavo` instead of `céntimo`? This should be covered by a test.

Reply:
Original base method uses CURRENCY_FORMS to set this words:
https://github.com/OCA/account-payment/pull/387/files#diff-eff0847a9b2281d7070165ce134fa319d9d11497af66581c82931c8deb70446eR13-R15

Reply:
OK, can you please add a test to ensure results?

Reply:
I was working on it. Test added.